packages/core/framework/package.json

```diff
@@ -10,6 +10,9 @@
     "!dist/**/__mocks__",
     "!dist/**/__fixtures__"
   ],
+  "bin": {
+    "medusa-db": "./dist/mikro-orm-cli/bin.js"
```

Comment:
thought: since this is specific to MikroORM, maybe we could call it `medusa-mikro-orm`

That said, maybe this is not super important, as this would not be used by end-users right?

Reply:
Yeah, we can call it `medusa-mikro-orm` too. I think, the end-users will use it if they develop custom modules. Because, we do not have any dedicated CLI for that

Reply:
> I think, the end-users will use it if they develop custom modules

Not in their own project right? We do have a CLI for managing migrations here afaik. 

Maybe we are talking about two different use cases.

Reply:
> Not in their own project right? We do have a CLI for managing migrations here afaik.

Yeah, the projects (aka starter kits) are all good, since they use the `npx medusa` CLI.

But custom modules will be created as we create them.

Reply:
Cool, that makes sense 👍